src/groups/mqb/mqba/mqba_clientsession.cpp

```diff
@@ -747,14 +747,32 @@ void ClientSession::onHandleConfiguredDispatched(
     // PRECONDITIONS
     BSLS_ASSERT_SAFE(dispatcher()->inDispatcherThread(this));
 
+    bdlma::LocalSequentialAllocator<2048> localAlloc(d_state.d_allocator_p);
+
+    const unsigned int qId =
+        streamParamsCtrlMsg.choice().isConfigureQueueStreamValue()
+            ? streamParamsCtrlMsg.choice().configureQueueStream().qId()
+            : streamParamsCtrlMsg.choice().configureStream().qId();
+
+    ClientSessionState::QueueStateMap::iterator queueStateIter =
+        d_queueSessionManager.queues().find(qId);
+    if (queueStateIter != d_queueSessionManager.queues().end()) {
+        d_logOpStream << "Configure queue '"
+                      << queueStateIter->second.d_handle_p->queue()->uri()
+                      << "'";
+    }
+    else {
+        d_logOpStream << "Configure queue [qId=" << qId << "]";
+    }
+
     if (isDisconnected()) {
         // The client is disconnected or the channel is down
+        logOperationTime();
```

Comment:
I think it was good to just pass the stream argument as before, even if we store it as a field now. Now the usage of `logOperationTime` is a bit obscure, it relies on a hidden dependency on `d_logOpStream`.

With argument passed, this dependency is shown in the interface.

Reply:
I am thinking if it's worth it to rename `d_logOpStream` to `d_currentOpDescription` to make it more clear

Reply:
Added argument to `logOperationTime`, renamed to `d_currentOpDescription`.